src/ape/managers/project.py

```diff
@@ -42,41 +86,128 @@ def sources(self) -> List[Path]:
 
         return files
 
+
+    def _load_contracts(self) -> Dict[str, ContractType]:
+        # Load a cached or clean manifest (to use for caching)
+        manifest = self.cached_manifest or PackageManifest()
```

Comment:
```suggestion
    def load_contracts(self, use_cache: bool=True) -> Dict[str, ContractType]:
        # Load a cached or clean manifest (to use for caching)
        manifest = use_cache and self.cached_manifest or PackageManifest()
```

NOTE: Can make use of this in the compile CLI